sdk/servicebus/azure-servicebus/azure/servicebus/aio/_async_auto_lock_renewer.py

```diff
@@ -72,8 +72,11 @@ def _renewable(self, renewable: Union[ReceivedMessage, ServiceBusSession]) -> bo
             return False
         if renewable._lock_expired:
             return False
-        if not renewable._receiver._running:
-            return False
+        try:
+            if not renewable._receiver._running: # type: ignore
+                return False
+        except AttributeError: # If for whatever reason the renewable isn't hooked up to a receiver
+            raise ServiceBusError("Cannot renew an entity without an associated receiver")
```

Comment:
see my question above, in what case would a ReceivedMessage/Session not be associated with a receiver?

Reply:
This is defensive; in case someone were to pass a standard Message or something else, to give a cogent error. (Could frankly be even more precise, specifying the expected classes here.)